There are some failures for `diagnostic-remote-logging` on Windows:

~~~

 │ │   influxdb.cpp
 │ │   node_main_influx.cpp
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\duration.hpp(109,3): error C2059: syntax error: '(' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\duration.hpp(109,3): error C2143: syntax error: missing ',' before '?' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\duration.hpp(109,3): error C2059: syntax error: '?' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\duration.hpp(159,3): error C2535: 'rclcpp::Duration::Duration(void)': member function already defined or declared [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │       %PREFIX%\Library\include\rclcpp\rclcpp\duration.hpp(107,3):
 │ │       see declaration of 'rclcpp::Duration::Duration'
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\time.hpp(204,3): error C2059: syntax error: '(' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\time.hpp(204,3): error C2143: syntax error: missing ',' before '?' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\time.hpp(204,3): error C2059: syntax error: '?' [%SRC_DIR%\build\influx_component.vcxproj]
 │ │   (compiling source file '../ros-jazzy-diagnostic-remote-logging/src/work/src/influxdb.cpp')
 │ │   
 │ │ %PREFIX%\Library\include\rclcpp\rclcpp\experimental\subscription_intra_process_base.hpp(169,39): error C2589
~~~

if you are not interested in Windows, can you move the failing package in the list of packages not built for Windows, i.e. https://github.com/avanmalleghem/ros-jazzy/blob/add-some-packages/vinca.yaml#L244-L255 ?  Thanks!

--- a/vinca.yaml
+++ b/vinca.yaml
@@ -182,7 +182,6 @@ packages_select_by_deps:
   - rtest
   
   - nmea-navsat-driver
-  - diagnostic-remote-logging
 
   # These packages are only built on Linux as they depend on Linux-specific API
   - if: linux
@@ -244,6 +243,7 @@ packages_select_by_deps:
   # These packages are currently not build on Windows, but they be with some work
   - if: not win
     then:
+      - diagnostic-remote-logging
       - plotjuggler-ros
       - foxglove_compressed_video_transport
       - rplidar_ros

